plugin/editorconfig.vim

```diff
@@ -24,12 +24,8 @@
 " POSSIBILITY OF SUCH DAMAGE.
 "
 
-if v:version < 700
-    finish
-endif
-
 " check whether this script is already loaded
```

Comment:
Could you also update this comment line?

Reply:
Thanks for your refview.
I updated comment.